.github/workflows/_reusable-verify-run-status.yaml

```
@@ -0,0 +1,47 @@
+# Prefix with "~" to sort last in Actions list
+name: ~Templates - Verify Run Status
+
+on:
+  workflow_call:
+    inputs:
+      check_run_name:
+        description: Name of the check run to verify
+        required: true
+        type: string
+      workflow_name:
+        description: Name of the workflow to verify
+        required: true
+        type: string
+
+permissions:
+  checks: read
+
+jobs:
+  check-run-status:
+    runs-on: ubuntu-24.04
+
+    steps:
+      # *** IMPORTANT ***
+      # For workflows that are triggered by the pull_request_target event, the workflow runs in the
+      # context of the base of the pull request.  You should make sure that you do not check out,
+      # build, or run untrusted code from the head of the pull request.
+      - uses: actions/checkout@v4
+        with:
+          # Only needs .github folder for automation, not the files in the PR (analyzed in a
+          # separate workflow).
+          #
+          # Uses the .github folder from the PR base branch (pull_request_target trigger),
+          # or the repo default branch (other triggers).
+          sparse-checkout: |
+            .github
+
+      - name: Verify matching status
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const { verifyRunStatus } = await import('${{ github.workspace }}/.github/workflows/src/verify-run-status.js');
+            return await verifyRunStatus(
+              { github, context, core }, 
+               "${{ inputs.check_run_name }}", 
+               "${{ inputs.workflow_name }}"
+            );
```

```suggestion
        uses: actions/github-script@v7
        env:
          CHECK_RUN: '${{ inputs.check_run_name }}'
          WORKFLOW: '${{ inputs.workflow_name }}'
        with:
          script: |
            const { verifyRunStatus } = await import('${{ github.workspace }}/.github/workflows/src/verify-run-status.js');
            return await verifyRunStatus({ github, context, core });
```

I believe this is safer, to prevent possible script injection.